templates/repo/projects/view.tmpl

```diff
@@ -77,13 +77,13 @@
 	</div>
 	<div class="ui container fluid padded" id="project-board">
 
-		<div class="board">
-			{{range $board := .Boards}}
+		<div class="project-board">
```

Comment:
project-column ?

Reply:
I already use `project-column` in line 83 for an actual column. This is the `div` containing all columns, so I guess `project-board` is to wrong.